ide/mercurial/src/org/netbeans/modules/mercurial/ui/repository/RepositoryPanel.java

```diff
@@ -227,7 +227,7 @@ private void chooseFolderButtonActionPerformed(java.awt.event.ActionEvent evt) {
         } catch (URISyntaxException | IllegalArgumentException ex) {
             //
         }
-        JFileChooser fileChooser = new AccessibleJFileChooser(NbBundle.getMessage(RepositoryPanel.class, "RepositoryPanel.FileChooser.Descritpion"), //NOI18N
```

Comment:
ide/git/src/org/netbeans/modules/git/ui/repository/remote/RemoteRepository.java:502

Reply:
I am new to open source contribution. I am not able to find the above file in my local project. Was this file added after I cloned the project?

Reply:
https://github.com/apache/netbeans/blob/master/ide/git/src/org/netbeans/modules/git/ui/repository/remote/RemoteRepository.java#L502

Reply:
I just checked that in my local repo I have changed it to 'Description' in the file but it is not reflecting in my remote repo file.

Reply:
What can I do now? Whether I have to again push my local repo to remote and make another pull request?

Reply:
@AanshSavla 
https://github.com/AanshSavla/netbeans/blob/typosCorrection/java/java.navigation/src/org/netbeans/modules/java/navigation/ElementNode.java#L312